xbmc/settings/lib/SettingsManager.cpp

```diff
@@ -512,7 +512,9 @@ void* CSettingsManager::GetSettingOptionsFiller(SettingConstPtr setting)
   // check if such an option filler is known
   auto fillerIt = m_optionsFillers.find(filler);
   if (fillerIt == m_optionsFillers.end())
+  {
     return nullptr;
+  }
```

Comment:
Looks unneeded.

Reply:
leftover from testing. dropped this change now